### PR TITLE
Basic styling and layout

### PR DESCRIPTION
**Pull request by n-wach**
Next step is to add some info about ME!

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,51 @@
...
+    height: 40px;
+}
+
+#about {
+  width: min(100% - 100px, 50%);
+  margin: 50px max(50px, 25%);
+  font-size: 20px;
+  text-align: start;
+}
+
```
Comment (sahilgoel) [inline portfolio/src/main/webapp/style.css:51]: Usually it is better to keep this file sorted both external fields as well as internal fields.
#about {
  font-size:
  margin:
  text-align:
  width:
}

body {
}

.
.

Comment (n-wach) [reply portfolio/src/main/webapp/style.css]: Fixed, thanks!

**Comment by n-wach** [comment]
Let me know if this works:

https://screenshot.googleplex.com/KC8mShsEG4A

**Comment by billie-cat** [comment]
> Let me know if this works:
> 
> https://screenshot.googleplex.com/KC8mShsEG4A

Worked! Thanks

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
+    <a class="social" href="https://github.com/n-wach">
+      <img src="images/github64.png" alt="Github">
+    </a>
+    <a class="social" href="https://linkedin.com/in/nathan-wachholz">
+      <img src="images/linkedin64.png" alt="LinkedIn">
+    </a>
+  </div>
+  <div id="about">
+    <p>
+      I'm from Santa Barbara, California, and am currently attending UCSB and
```
Comment (f-camp) [inline portfolio/src/main/webapp/index.html:22]:
```suggestion
      I'm from Santa Barbara, California, and I'm currently attending UCSB and
```

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,42 @@
...
+      also a STEP intern at Google for the Summer of 2020.
+    </p>
+    <p>
+      Outside of school and work, I'm a huge fan of the outdoors.  I love
+      hiking, biking, and the beach.  The picture above is from a hike to
+      Inspiration Point in Santa Barbara.  I also recently picked up gardening!
+      I'm a big reader, and play my fair share of video games (and board games!).
+    </p>
+    <p>
+      Below, I've included a selection of exciting projects, pictures, and
```
Comment (f-camp) [inline portfolio/src/main/webapp/index.html:37]: nit: How will you attach the links to the pictures and projects? 
If you haven't done so yet maybe you can add a comment signifying a TODO. This means it's not yet there but it's on the way.
Could be something like: <! – – TODO(nwach): Add projects, pictures and facts -->

Comment (n-wach) [reply portfolio/src/main/webapp/index.html:37]: I've already started working on the links/projects section--should I still go back and add/commit a comment?

I'll start doing these comments in the future though, thanks!

Comment (f-camp) [reply portfolio/src/main/webapp/index.html:37]: This is ok for now, in the future if you haven't committed the code for a feature you might be referring to its good practice to add a TODO so one might not be confused if the feature is not there after your commit.

Comment (n-wach) [reply portfolio/src/main/webapp/index.html:37]: Makes sense :)

**Comment by billie-cat** [comment]
Hey Nathan,
Please wait until Fola and I have both approved the change before merging, so we both have a chance to look it over. Not a big deal for this one, but keep in mind for future ones. Typically at Google you only need 1 person's approval, but given its an internship and we are both writing your evaluation it helps us to be able to look over all your code.

**Comment by n-wach** [comment]
Oh sorry about that!  Will do so in the future

**Comment by billie-cat** [comment]
Thanks!

**Pull request merged**